bin/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -xe
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+function run-gunicorn () {
+    if [[ -z "$NEW_RELIC_LICENSE_KEY" ]]; then
+        exec gunicorn "$@"
+    else
+        export NEW_RELIC_CONFIG_FILE=newrelic.ini
+        exec newrelic-admin run-program gunicorn "$@"
+    fi
+}
+
+# look for the required files and fail quickly if it's not there
+STARTUP_FILES=(
+    "birdbox/data/basket/basket.mozilla.org.json"
```

Comment:
missing a comma here? Or did you want one concatenated string?

Reply:
Oh, this isn't Python. So maybe all good?

Reply:
Yeah, I think it's fine - Bedrock has the same:

https://github.com/mozilla/bedrock/blob/ac82b6b46367eaa271e8c5a7bac427867ad1c07b/bin/run-prod.sh#L17-L21